odra-macros/src/utils/ident.rs

```diff
@@ -35,6 +35,14 @@ pub fn init() -> syn::Ident {
     format_ident!("init")
 }
 
+pub fn epc() -> syn::Ident {
```

Comment:
What are `epc` and `load`?

Reply:
New elements of Deployer - epc is extracted method for generating entry point calls, load loads already deployed contract, but registers it's epcs